pkg/schema/event_parser.go

```diff
@@ -58,17 +58,15 @@ func GetEvent(name string) (*EventSchema, error) {
 }
 
 func IsEventDisabled(name string) bool {
-	controllerFilePath := "/events/event-schema/controller.yaml"
-	workerFilePath := "/events/event-schema/worker.yaml"
-	controllerConfigs, err := parseConfig(controllerFilePath)
-	if err != nil {
-		return false
-	}
-	workerConfigs, err := parseConfig(workerFilePath)
+	return isDisabledInFile(name, "/events/event-schema/controller.yaml") ||
+		isDisabledInFile(name, "/events/event-schema/worker.yaml")
+}
+
+func isDisabledInFile(name string, filepath string) bool {
+	configs, err := parseConfig(filepath)
```

Comment:
why do we need two separate files for disabling events? can't we use the same path in both controller and worker?

Reply:
We can do that. Will discuss with @richiesebastian and make the change.